api/data/filters/filters.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "name": "countries",
+    "isCustomFilter": false,
+    "values": [
+      "Austria",
+      "Belgium",
+      "Bulgaria",
+      "Croatia",
+      "Cyprus",
+      "Czechia",
+      "Denmark",
+      "Estonia",
+      "Finland",
+      "France",
+      "Germany",
+      "Greece",
+      "Hungary",
+      "Ireland",
+      "Italy",
+      "Latvia",
+      "Lithuania",
+      "Luxembourg",
+      "Malta",
+      "Poland",
+      "Portugal",
+      "Romania",
+      "Slovakia",
+      "Slovenia",
+      "Spain",
+      "Sweden",
+      "The Netherlands"
+    ],
+    "label": {
+      "selected": "Country is",
+      "unSelected": "All Countries"
```

Comment:
Maybe we are missing one extra label for when multiple values are selected?

Smth like: `Countries are...`

But not sure

Reply:
@alexeh @alepefe-vizz I have already checked this with design, and the selected label is always in singular form.